doc/index.md

```diff
@@ -1010,13 +1012,25 @@ sdk.on('pendingSession', (pendingSession: IPendingSession) => { });
 Value of event:
 ``` ts
 interface IPendingSession {
```

Comment:
Wait, shouldn't we be using the PendingSession from the streaming client? I thought we changed to that recently or maybe I just mentioned it.